src/pages/FormOne/FormOne.scss

```diff
@@ -0,0 +1,472 @@
+.event-container {
+	display: flex;
+	z-index: 3;
+	height: 100%;
+	flex-wrap: nowrap;
+
+	.logo-talkien {
+		right: 1150px;
+		top: -300px;
+		width: 80%;
+		object-fit: contain;
+		opacity: 0.1;
+		position: absolute;
+		z-index: -1;
+	}
+
+	.right-block {
+		width: 100%;
+		height: 100%;
+		text-align: left;
+		overflow: auto;
+		z-index: 3;
+	}
+
+	.block-event {
+		margin: 4.5% 4.5%;
+		height: 100%;
+	}
+
+	.input-event-name {
+		input {
+			font-family: Roboto;
+			font-size: 36px;
+			font-weight: normal;
+			font-style: normal;
+			font-stretch: normal;
+			line-height: normal;
+			letter-spacing: 0.46px;
+			border-image-slice: 1;
+		}
+	}
+
+	.date-and-place {
+		width: 100%;
+		display: flex;
+		margin-top: 2.1%;
+		justify-content: space-between;
+		flex-wrap: wrap;
+	}
+
+	.start,
+	.end,
+	.places {
+		display: flex;
+		justify-content: space-between;
+		align-items: center;
+		flex-wrap: nowrap;
+		width: 33%;
+	}
+
+	.input-startDate,
+	.input-endDate,
+	.input-address {
+		input {
+			font-family: Roboto;
+			font-size: 24px;
+			font-weight: normal;
+			font-style: normal;
+			font-stretch: normal;
+			line-height: normal;
+			letter-spacing: 0.3px;
+			color: #ffffff;
+		}
+	}
+
+	.from,
+	.to,
+	.in {
+		font-family: Roboto;
+		font-size: 24px;
+		font-weight: normal;
+		font-style: normal;
+		font-stretch: normal;
+		line-height: normal;
+		letter-spacing: 0.3px;
+		color: #ffffff;
+		margin-right: 2%;
+	}
+
+	.fullAddress-inputs {
+		margin-right: 32px;
+		width: 100%;
+	}
+
+	.adress {
+		display: flex;
+		height: fit-content;
+		margin-top: 2.7%;
+	}
+
+	.input-full-address-of-the-place,
+	.input-longitude,
+	.input-latitude {
+		label {
+			margin-top: 2.7%;
+		}
+		input {
+			font-family: Roboto;
+			font-size: 24px;
+			font-weight: normal;
+			font-style: normal;
+			font-stretch: normal;
+			line-height: normal;
+			letter-spacing: 0.3px;
+			color: #ffffff;
+		}
+	}
+
+	.input-short-descriptions {
+		label {
+			margin-top: 2.7%;
+		}
+		input {
+			border-color: #f0f0f0;
+			font-size: 24px;
+			border-width: 2px;
+			border-top: 0px;
+			border-left: 0px;
+			border-right: 0px;
+			background: transparent;
+		}
+	}
+
+	.textarea-long-description {
+		label {
+			margin-top: 2.7%;
+		}
+		textarea {
+			width: 100%;
+			height: 128px;
```

Comment:
Don't force textarea height. Instead, use `numberOfLines` html property to define base size